frontend/src/components/ProfileEditReviewer.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <ReviewerCard
+    v-if="reviewer"
```

Comment:
there should be a better indication that the reviewer record is inactive here. Currently it just shows the "activate" button. Maybe a badge for `active`/`inactive` on the reviewer card card-title in general and possibly hiding everything besides a `Reactivate` button here on the profile

Reply:
![image](https://github.com/user-attachments/assets/f55ab281-8273-41e9-8554-3231d69d6191)
Does something like this work?

Reply:
yeah that works, good call.

Since the edit is hidden we can use the regular blue link color, and maybe change the icon + wording

```html
        <a v-else class="btn btn-link" @click="changeActiveState(reviewer, true)">
          <small><i class="fas fa-sync-alt me-1"></i>Reactivate</small>
        </a>
```